cmd/commands/cli/command_identities.go

```diff
@@ -569,3 +572,34 @@ func (c *cliApp) importIdentity(actionsArgs []string) (err error) {
 	clio.Success("Identity imported:", id.Address)
 	return nil
 }
+
+const usageLastWithdrawal = "last-withdrawal <identity>"
+
+func (c *cliApp) lastWithdrawal(actionArgs []string) error {
+	if len(actionArgs) != 1 {
+		clio.Info("Usage: " + usageGetReferralCode)
+		return errWrongArgumentCount
+	}
+
+	address := actionArgs[0]
+	history, err := c.tequilapi.SettleHistory(address)
+	if err != nil {
+		return fmt.Errorf("Could not get last withdrawal: %w", err)
+	}
+	if history.TotalItems == 0 {
+		clio.Info("No withdrawals found")
+		return nil
+	}
+
+	lastWithdrawal := history.Items[0]
+
+	clio.Info("Tx hash: ", lastWithdrawal.TxHash)
+	clio.Info("DateTime: ", lastWithdrawal.SettledAt)
+	clio.Info("Amount: ", money.New(lastWithdrawal.Amount))
+	clio.Info("Fees: ", money.New(lastWithdrawal.Fees))
+	clio.Info("Blockchain explorer Url: ", lastWithdrawal.BlockExplorerURL)
+	if lastWithdrawal.Error != "" {
```

Comment:
Maybe it's just me, but I would rather always print the Error field but let it be `null` or something like that if no error exists. It would be easier to parse if someone is using the CLI in a script of some sort 🤷🏻 

Though this I guess would be up for debate.

Reply:
I agree